extensions/blocks/map/edit.js

```diff
@@ -185,6 +270,37 @@ class MapEdit extends Component {
 						] }
 					/>
 					<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
+						<BaseControl
+							label={ __( 'Height in pixels' ) }
+							id={ `block-jetpack-map-height-input-${ instanceId }` }
+						>
+							<input
```

Comment:
I wonder if we could style this a bit better (this is in WP 5.3):

![image](https://user-images.githubusercontent.com/426388/74763644-77f41d80-5280-11ea-8c1f-2b7197756d97.png)

It looks better in 5.2 though:

![image](https://user-images.githubusercontent.com/426388/74765023-1aad9b80-5283-11ea-9619-a187ec25c68e.png)

Reply:
👍🏻 8ea23b5a6